lib/commands/command-info.bash

```diff
@@ -4,9 +4,13 @@
 
 info_command() {
   printf "%s:\n%s\n\n" "OS" "$(uname -a)"
-  printf "%s:\n%s\n\n" "SHELL" "$($SHELL --version)"
+  printf "%s:\n%s\n\n" "SHELL" "Bash $BASH_VERSION"
   printf "%s:\n%s\n\n" "ASDF VERSION" "$(asdf_version)"
-  printf "%s:\n%s\n\n" "ASDF ENVIRONMENT VARIABLES" "$(env | grep -E "ASDF_DIR|ASDF_DATA_DIR|ASDF_CONFIG_FILE|ASDF_DEFAULT_TOOL_VERSIONS_FILENAME")"
+  printf '%s\n' 'ASDF ENVIRONMENT VARIABLES:'
+  printf 'ASDF_DEFAULT_TOOL_VERSIONS_FILENAME=%s\n' "${ASDF_DEFAULT_TOOL_VERSIONS_FILENAME-(not set)}"
```

Comment:
Although not actually set, the default is `.tool-versions`, so printing that it isn't set might be confusing. I guess this could be raised as a separate issue to actually set `ASDF_DEFAULT_TOOL_VERSIONS_FILENAME` to the default `.tool-versions`.

Reply:
You bring up a good point - these variables are labeled under `ENVIRONMENT VARIABLES:`, yet they may not be env vars set by the user (only internally by asdf). And, I think like you mention, `ASDF_DEFAULT_TOOL_VERSIONS_FILENAME` isn't set internally by asdf like the others, causing more confusion.

How does it sound to print `INTERNAL VARIABLES: `, to make it clear that these variables aren't the same values as the environment variables the user passed in? In a separate issue I can set `ASDF_DEFAULT_TOOL_VERSIONS_FILENAME` like we do with the other variables

Reply:
That sounds like a good path forward, thanks.